include/zephyr/drivers/video-controls.h

```diff
@@ -107,18 +227,307 @@ enum video_power_line_frequency {
+enum video_auto_focus_status {
+	VIDEO_AUTO_FOCUS_STATUS_IDLE = 0,
+	VIDEO_AUTO_FOCUS_STATUS_BUSY = BIT(0),
+	VIDEO_AUTO_FOCUS_STATUS_REACHED = BIT(1),
+	VIDEO_AUTO_FOCUS_STATUS_FAILED = BIT(2),
+};
+
+/** Determines auto focus distance range for which lens may be adjusted. */
+#define VIDEO_CID_AUTO_FOCUS_RANGE (VIDEO_CID_CAMERA_CLASS_BASE + 31)
+enum video_auto_focus_range {
+	VIDEO_AUTO_FOCUS_RANGE_AUTO = 0,
+	VIDEO_AUTO_FOCUS_RANGE_NORMAL = 1,
+	VIDEO_AUTO_FOCUS_RANGE_MACRO = 2,
+	VIDEO_AUTO_FOCUS_RANGE_INFINITY = 3,
+};
+
+/**This control turns the camera horizontally at the specific speed.
+ * The unit is undefined. A positive value moves the camera to the right (clockwise when viewed
+ * from above), a negative value to the left. A value of zero stops the motion if one is in
+ * progress and has no effect otherwise.
+ */
+#define VIDEO_CID_PAN_SPEED (VIDEO_CID_CAMERA_CLASS_BASE + 32)
+
+/** This control turns the camera vertically at the specified speed.
+ * The unit is undefined. A positive value moves the camera up, a negative value down.
+ * A value of zero stops the motion if one is in progress and has no effect otherwise.
+ */
+#define VIDEO_CID_TILT_SPEED (VIDEO_CID_CAMERA_CLASS_BASE + 33)
+
+/** This read-only control describes the camera position on the device
+ * It by reports where the camera camera is installed, its mounting position on the device.
+ * This control is particularly meaningful for devices which have a well defined orientation,
+ * such as phones, laptops and portable devices since the control is expressed as a position
+ * relative to the device's intended usage orientation.
+ * , or , are said to have the
+ * @ref VIDEO_CAMERA_ORIENTATION_EXTERNAL orientation.
+ */
+#define VIDEO_CID_CAMERA_ORIENTATION (VIDEO_CID_CAMERA_CLASS_BASE + 34)
+enum video_camera_orientation {
+	/** Camera installed on the user-facing side of a phone/tablet/laptop device */
+	VIDEO_CAMERA_ORIENTATION_FRONT = 0,
+	/** Camera installed on the opposite side of the user */
+	VIDEO_CAMERA_ORIENTATION_BACK = 1,
+	/** Camera sensors not directly attached to the device or that can move freely */
+	VIDEO_CAMERA_ORIENTATION_EXTERNAL = 2,
+};
+
+/** This read-only control describes the orientation of the sensor in the device.
+ * The value is the rotation correction in degrees in the counter-clockwise direction to be
+ * applied to the captured images once captured to memory to compensate for the camera sensor
+ * mounting rotation.
+ */
+#define VIDEO_CID_CAMERA_SENSOR_ROTATION (VIDEO_CID_CAMERA_CLASS_BASE + 35)
+
+/** Change the sensor HDR mode.
+ * A HDR picture is obtained by merging two captures of the same scene using two different
+ * exposure periods. HDR mode describes the way these two captures are merged in the sensor.
+ * As modes differ for each sensor, menu items are not standardized by this control and have
+ * hardware-specific values.
+ */
+#define VIDEO_CID_HDR_SENSOR_MODE (VIDEO_CID_CAMERA_CLASS_BASE + 36)
```

Comment:
REMOVE? Used in only one Linux driver:
drivers/media/i2c/st-vgxy61.c
Also, hello Alain Volmat! https://lwn.net/Articles/910842
Would this control be useful to have around?